### DISCUSSION TITLE
Qt client, the upload file option is not Unavailable when your not connected to any server:

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1575] [Ideas]
### Description

when your disconnected and your not connected to any server, in the teamtalk window, go to files tab and files list, press application, then you will see it will say (Upload File  Alt+l). and when you press enter, it will back to files list, and nothing happens: this option should become (Upload File  unavailable  Alt+l) when your not connected to any server, as how download file and delete file options is unavailable when there is no file to download and no file to delete, so this upload file option also should work as those other ones: Qt client vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
Please describe steps to reproduce. I don't understand the explanation.

**Comment by amirmahdifard** [comment]
@bear101 so this is clear that you can't upload any file if you didn't connected to any server right? 1: just open the teamtalk but don't connect to any server. 2: go to files list and press application to open the context menu, then you will see the difrents between the download file and delete file with upload file option: the upload file option should be unavailable in there because you can't upload a file if you not connected to any server:  can you?
